var/spack/repos/builtin/packages/hpx-kokkos/package.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```

Comment:
```suggestion
# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```
```suggestion
# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```

Reply:
Not sure why the suggestion is repeated here.  Took the liberty of automatically merging it since it is a minor change.